i18n/en-US/common.json

```diff
@@ -0,0 +1,55 @@
+{
+    "REQUEST_ERROR": "An error occurred making that request.",
+    "INVALID_OPTION": "Invalid option given.",
+    "NONE": "None",
+    "USER_FETCH_ERROR": "An error occured while trying to fetch the requested user.",
+    "ALERT_SYSTEM": "TypicalBot Alert System",
+    "MODERATOR_FIELD": "» Moderator",
+    "REASON_FIELD": "» Reason",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.